openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSettingLeftPanel.test.tsx

```diff
@@ -0,0 +1,476 @@
+/*
+ *  Copyright 2023 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { render, screen, waitForElement } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+import { MemoryRouter, Route } from 'react-router-dom';
+import GlobalSettingLeftPanel from './GlobalSettingLeftPanel';
+
+jest.mock('../PermissionProvider/PermissionProvider', () => ({
+  usePermissionProvider: jest.fn().mockImplementation(() => ({
+    permissions: {
+      alert: {
+        Create: true,
+        Delete: true,
+        ViewAll: true,
+        EditDescription: true,
+      },
+      alertAction: {
+        Create: true,
+        Delete: true,
```

Comment:
Can we move this to a mock file, Taking lots of space in test file

Reply:
Sure, will do that.